contracts/mainnet/connectors/morpho-compound/helpers.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+import "./interface.sol";
+import "../../common/stores.sol";
+import "../../common/basic.sol";
+import "../../common/interfaces.sol";
+
+abstract contract Helpers is Stores, Basic {
+	IMorphoCore public constant morphoCompound =
```

Comment:
```suggestion
	IMorphoCore public constant MORPHO_COMPOUND =
```

Just to follow solidity guidelines. Same comment for every constant variable.

Reply:
Got it! Will keep this in mind.